app/controllers/v0/benefits_claims_controller.rb

```diff
@@ -305,15 +309,9 @@ def suppress_evidence_requests(claim)
       claim
     end
 
-    def report_evidence_submission_metrics(endpoint, claim_ids)
-      claim_ids = Array(claim_ids) # Ensure it's always an array
+    def report_evidence_submission_metrics(endpoint, evidence_submissions)
+      status_counts = evidence_submissions.group_by(&:upload_status).transform_values(&:count)
```

Comment:
Vets-API has a large database, are you sure you want to remove this query from the ActiveRecord level?